doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2725 @@
+
+**Input:**
+
+::
+
+  array = np.ones(2, dtype=int64) 
+  array
+
+**Output:**
+
+::
+
+  array([1, 1])
+
+`Learn more about creating arrays here
+<https://numpy.org/devdocs/user/quickstart.html#array-creation>`_.
+
+Adding, removing, and sorting elements 
+--------------------------------------
+
+
+::
+
+  np.sort() 
+  np.concatenate()
+
+-----
+
+
+Sorting an element is simple with ``np.sort()``. You can specify the axis, kind,
+and order when you call the function. 
+`Read more about sorting an array here <https://numpy.org/devdocs/reference/generated/numpy.sort.html>`_.
+
+If you start with this array:
+
+::
+
+  arr = np.array([2, 1, 5, 3, 7, 4, 6, 8])
+
+You can quickly sort the numbers in ascending order with:
+
+**Input:**
+
+::
+
+  np.sort(arr)
+
+**Output:**
+
+::
+
+  array([1, 2, 3, 4, 5, 6, 7, 8])
+
+In addition to sort, which returns a sorted copy of an array, you can use:
+
+``argsort``, which is an 
+`indirect sort along a specified axis <https://numpy.org/devdocs/reference/generated/numpy.argsort.html#numpy.argsort>`_,
+``lexsort``, which is an 
+`indirect stable sort on multiple keys <https://numpy.org/devdocs/reference/generated/numpy.lexsort.html#numpy.lexsort>`_,
+``searchsorted``, which will 
+`find elements in a sorted array <https://numpy.org/devdocs/reference/generated/numpy.searchsorted.html#numpy.searchsorted>`_,
+and  ``partition``, which is a 
+`partial sort  <https://numpy.org/devdocs/reference/generated/numpy.partition.html#numpy.partition>`_.
+
+If you start with these arrays:
+
+::
+
+  a = np.array([1, 2, 3, 4]) 
+  b = np.array([5, 6, 7, 8])
+ 
+
+You can concatenate them with ``np.concatenate()``. 
+
+**Input:**
+
+::
+
+  np.concatenate((a, b))
+
+**Output:**
+
+::
+
+  array([1, 2, 3, 4, 5, 6, 7, 8])
+
+Or, if you start with these arrays:
+
+::
+
+  x = np.array([[1, 2], [3, 4]]) 
+  y = np.array([[5, 6]])
+
+You can concatenate them with:
+
+
+::
+
+  np.concatenate((x, y), axis=0)
+
+**Output:**
+
+::
+
+  array([[1, 2], 
+         [3, 4], 
+         [5, 6]])
+
+
+In order to remove elements from an array, it's simple to use indexing to select
+the elements that you want to keep.
+
+`Read more about concatenate here <https://numpy.org/devdocs/reference/generated/numpy.concatenate.html#numpy.concatenate>`_
+.
+
+How do you know the shape and size of an array?
+-----------------------------------------------
+
+
+::
+
+  ndarray.ndim()  
+  ndarray.size() 
+  ndarray.shape()
+
+-----
+
+**ndarray.ndim** will tell you the number of axes, or dimensions, of the array.
+
+**ndarray.size** will tell you the total number of elements of the array. This
+**is the *product* of the elements of the array's shape.
+
+**ndarray.shape** will display a tuple of integers that indicate the number of
+**elements stored along each dimension of the array. If, for example, you have a
+**2D array with 2 rows and 3 columns, the shape of your array is (2,3).
+
+For example, if you create this array:
```

Comment:
Something has gone wrong here, it looks like your wrapping tool has converted your emphasis to a list.

My preference would be not to hard-wrap sentences, it just makes editing harder, and leads to struggles with tools like this one, without any real gain IMO.

Reply:
Got it